hotwax/dxp-components
# Implemented: centralized OMS instance navigation component in dxp-components (#140)

Related Issues #140, #121

### Short Description and Why It's Useful

Currently all apps used its own code for OMS instance navigation in settings. Here's implemented a centralized OMS instance navigation which can be used by any apps.

- Created a component in dxp-components for the OMS instance navigation.
- Used "goToOms" method of the dxp-component instead of using of app. This also solves 'goToOms' (#121).
- Checked the proper working of this implementation in job-manager app.

### Steps to verify
- Can verify the proper working of this implementation using job-manager PR raised in https://github.com/hotwax/job-manager/pull/610
- Review changes of job-manager PR https://github.com/hotwax/job-manager/pull/610

### Screenshots of Visual Changes before/after (If There Are Any)



**IMPORTANT NOTICE** - Remember to add changelog entry


### Contribution and Currently Important Rules Acceptance


- [x] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)
